src/Gemini.Modules.Inspector/ViewModels/InspectorViewModel.cs

```diff
@@ -44,5 +45,32 @@ private void RaiseSelectedObjectChanged()
             EventHandler handler = SelectedObjectChanged;
             if (handler != null) handler(this, EventArgs.Empty);
         }
+
+        public void ResetAll()
+        {
+            if (SelectedObject == null)
+                return;
+
+            RecurseEditors(SelectedObject.Inspectors, delegate(Inspectors.IEditor editor) {
```

Comment:
It's not a big deal, and don't worry about changing this one, but please could you put open braces on their own line?